frontend/src/pages/NewStudent.vue

```diff
@@ -132,37 +121,54 @@ export default {
     }
   },
 
-  computed: {
-    filteredLanguageDataArray() {
-        return this.languageData.filter((option) => {
-            return option
-              .toString()
-              .toLowerCase()
-              .indexOf(this.nativeLanguage.toLowerCase()) >= 0
-            })
-    },
-      filteredSourceDataArray() {
-        return this.sourceData.filter((sourceOption) => {
-            return sourceOption
-              .toString()
-              .toLowerCase()
-              .indexOf(this.source.toLowerCase()) >= 0
-            })
-    }  
+ async mounted() {
+    // Call API for Native Languages
+      await fetch("/api/nativeLanguages")
+        .then(response => response.json())
+        .then(result => this.API_nativeLanguage = result)
   },
 
-
   methods: {
     createStudent(){
+       const studentCreate = {
+        method: "POST",
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          PhoneNumber: this.PhoneNumber,
+          FirstName: this.name,
```

Comment:
You might as well just start calling this `FullName` instead of `FirstName`. Saves one round of refactoring later.